tensorflow/core/kernels/data/dataset_test_base.h

```diff
@@ -236,8 +236,6 @@ class BatchDatasetParams : public DatasetParams {
 
   string op_name() const override;
```

Comment:
@jsimsa Would it will be better to rename the function `op_name()` to `dataset_type()` as `op_name()` may make some confusing like [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/data/dataset_test_base.cc#L725):
* [name_utils::OpName(const string& dataset_type)](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/data/name_utils.h#L58) will return something like `MapDataset`
* `DatasetParams::op_name()` return the dataset type (e.g. `Map`).

Reply:
SGTM